packages/global/scss/index.scss

```diff
@@ -375,6 +375,13 @@ $theme-site-header-breakpoints: map-merge(
 }
 .card-deck-flow {
   .node {
+    .node {
+      &__title {
+        &--published {
```

Comment:
Semantically it would be better for the modifier to describe what's being modified. In this case bold would be preferred over published.